### DISCUSSION TITLE
Dockerizing the app

### DISCUSSION THREAD

**Discussion by hwangswan** [khongsomeo/hcmus-dstn#9] [Ideas]
Hi, 

Normally with applications that requires *installing dependencies before running* we should also make things easy by providing a Docker image. That way, users can run the app without shenanigans.

I'm consider adding a Docker image, which requires a pipeline to automatically build & push the docker image into ghcr. Looking for further supports.

**Comment by trhgquan** [comment]
Your previous PR has been merged. It's time for a workflow to build & publish the package automatically.

I am tagging @meadoge for this.

**Comment by meadoge** [comment]
It's easy to build one, but again, may cost us several commits to make it
works.
